code/datums/supplypacks/supplypack.dm

```diff
@@ -18,6 +20,10 @@
 	if(!num_contained)
 		for(var/entry in contains)
 			num_contained += max(1, contains[entry])
+	cost = 0
+	for(var/entry in contains)
+		cost += atom_info_repository.get_worth_for(entry) * max(1, contains[entry])
+	cost *= CARGO_PRICE_MARKUP
```

Comment:
Should add the value of the crate itself. For quite a few items that's actually the bulk of the cost, even currently.